server/services/nomisPrisonerService.test.ts

```diff
@@ -128,7 +128,7 @@ describe('NomisPrisonerService tests', () => {
     it('should return suspended allocations', async () => {
       fakeNomisPrisonerService
         .get('/allocations/suspended')
-        .query({ prisonId: 'BXI', excludeProgramCodes: ['SAA_EDUCATION', 'SAA_INDUSTRY'] })
+        .query({ prisonId: 'BXI', excludeProgramCode: ['SAA_EDUCATION', 'SAA_INDUSTRY'] })
```

Comment:
This never worked, but nobody ever ran the migration preview after a migration had started (until today) so nobody noticed.